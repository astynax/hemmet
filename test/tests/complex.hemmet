form:search-form$theme>input.query^red-text>(div.help~hidden_t)+span.submit&button~text_small>.hint
