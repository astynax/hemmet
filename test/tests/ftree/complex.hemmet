file1 folderA/{fileA1 folderAA/ folderAB/{fileAB1 fileAB2} fileA2} folderB/folderBA/folderBAA/fileBAA1 file2
