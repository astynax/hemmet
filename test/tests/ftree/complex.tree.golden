.
├── file1
├── file2
├── folderA/
│   ├── fileA1
│   ├── fileA2
│   ├── folderAA/
│   └── folderAB/
│       ├── fileAB1
│       └── fileAB2
└── folderB/
    └── folderBA/
        └── folderBAA/
            └── fileBAA1
